source/content.js

```diff
@@ -7,6 +7,7 @@ import codeHighlight from './features/code-highlight';
 import mentionHighlight from './features/mentions-highlight';
 import addLikesButtonNavBar from './features/likes-button-navbar';
 import keyboardShortcuts from './features/keyboard-shortcuts';
+import renderInlineCode from './features/markdown';
```

Comment:
Can we just name this something like 'inline-code' instead of 'markdown'? It's not actually markdown, since things like `**bold**` would be rendered as is and not converted to HTML.

Reply:
I was going to implement **bold** and _italics_ in this file too but I'll make this change.